Documentation/data-type-and-flow-control.md

```diff
@@ -116,13 +140,13 @@ offset |       0       |       1       |       2       |       3       |
        -----------------------------------------------------------------
 ```
 
-## other/tensorsave (TBU)
-
-```other/tensorsave```, along with its ```typefind``` definition, is defined to enable to save ```other/tensors``` streams as files and load such files are ```other/tensors``` stream. With the definitions of headers defined with ```other/tensorsave```, GStreamer can decode the given file and determine that the file belongs to ```other/tensorsave```.
-
-The detailed description of the file format is at [Design External Save Format for other/tensor and other/tensors Stream for TypeFind](https://github.com/nnstreamer/nnstreamer/wiki/Design-External-Save-Format-for-other-tensor-and-other-tensors-Stream-for-TypeFind)
+## other/tensors,format=sparse
 
+```other/tensors,format=sparse``` allows to express sparse tensors (tensors with a lot of zeros) efficiently (in terms of memory size) by extending ```other/tensors,format=flexible```.
+In general, nnstreamer elements (tensors-*) do not support sparse tensors unlike static and flexible, although a few tensor-filter subplugins support sparse tensors.
+Sparse tensors exist for efficient transmission and data storing.
 
+If your source data streams or sink data streams are to be in sparse tensors, you may apply tensor-converter to convert them from/to static or flexible tensors.
```

Comment:
I think this words be like " ... you may apply tensor_sparse_[enc|dec] to convert them from/to static tensors. "